packages/kbn-es/src/utils/wait_for_security_index.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { Client } from '@elastic/elasticsearch';
+import { ToolingLog } from '@kbn/tooling-log';
+const DEFAULT_READY_TIMEOUT = 10 * 1000; // 10 seconds
```

Comment:
Should we make this longer with the extended refresh interval?

Reply:
On second pass that doesn't look relevant to api key deletion, and the flaky test runner agrees.